site/docs/concepts/beam-search.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 3
+---
+
+# Beam search
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident.
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident.
+## Chapter one
```

Comment:
There's docs layout map on the left in https://yatarkan.github.io/openvino.genai/docs/overview/introduction#chapter-one and at the same time there's a page layout map on the right. Should they be merged?

Reply:
No, left menu shows sections and pages structure, the right one - current page sections. Actually, in OpenVINO docs there is the same UI pattern (e.g. https://docs.openvino.ai/2025/openvino-workflow-generative/inference-with-genai.html )